tests/multisigNFTs_test.go

```diff
@@ -0,0 +1,192 @@
+		{"multiSig", false, false, "[case-2.0] Create MultiSig Tx for NFTs [Burn-item] - [internalTx3x_burnItem_err1] - wait-15-seconds.", "multisig-acc-nft-2", "100000000cin", 0, MultisigInfo{"create-internal-tx", "", "", 0, nil, "nft-grp-addr-2", 5, internalTx3x_burnItem_err1}, "MEMO : xxxxx", nil}, // nx "sequence":
+		{"multiSig", false, false, "[case-2.0] Create MultiSig Tx for NFTs [Burn-item] - [internalTx3x_burnItem_err3] - wait-15-seconds.", "multisig-acc-nft-2", "100000000cin", 0, MultisigInfo{"create-internal-tx", "", "", 0, nil, "nft-grp-addr-2", 6, internalTx3x_burnItem_err3}, "MEMO : xxxxx", nil}, // nx "sequence":
+		{"multiSig", false, false, "[case-2.0] Sign MultiSig Tx for NFTs [Burn-item] - submit which signed by multisig-acc-nft-3 - wait-15-seconds.", "multisig-acc-nft-3", "100000000cin", 0, MultisigInfo{"multiSig-sign-tx", "", "", 0, nil, "nft-grp-addr-2", 4, internalTx3x_burnItem1}, "MEMO : Sign MultiSig Tx", nil},
+
+		//transferTokenOwnership - Verify - Accept
+		{"multiSig", false, false, "[case-2.0] Create MultiSig Tx for NFTs [Transfer-token-ownership] - submit counter+3 - Happy Path - wait-15-seconds.", "multisig-acc-nft-2", "100000000cin", 0, MultisigInfo{"create-internal-tx", "", "", 0, nil, "nft-grp-addr-2", 7, internalTx3x_transferTokenOwnership1}, "MEMO : xxxxx", nil},                  // nx "sequence":
+		{"multiSig", false, false, "[case-2.0] Create MultiSig Tx for NFTs [Transfer-token-ownership] - [internalTx3x_transferTokenOwnership_err1] - wait-15-seconds.", "multisig-acc-nft-2", "100000000cin", 0, MultisigInfo{"create-internal-tx", "", "", 0, nil, "nft-grp-addr-2", 8, internalTx3x_transferTokenOwnership_err1}, "MEMO : xxxxx", nil}, // nx "sequence":
+		{"multiSig", false, false, "[case-2.0] Create MultiSig Tx for NFTs [Transfer-token-ownership] - [internalTx3x_transferTokenOwnership_err3] - wait-15-seconds.", "multisig-acc-nft-2", "100000000cin", 0, MultisigInfo{"create-internal-tx", "", "", 0, nil, "nft-grp-addr-2", 9, internalTx3x_transferTokenOwnership_err3}, "MEMO : xxxxx", nil}, // nx "sequence":
+		{"multiSig", false, false, "[case-2.0] Sign MultiSig Tx for NFTs [Transfer-token-ownership] - submit which signed by multisig-acc-nft-3 - wait-15-seconds.", "multisig-acc-nft-3", "100000000cin", 0, MultisigInfo{"multiSig-sign-tx", "", "", 0, nil, "nft-grp-addr-2", 7, internalTx3x_transferTokenOwnership1}, "MEMO : Sign MultiSig Tx", nil},
+		{"nonFungibleToken", false, false, "[case-2.0] VERIFY nonfungible token transfer ownership [TNFT-PUBLIC-FALSE-02] - Happy path - wait-15-seconds.", "nft-mostafa", "0cin", 0, NonFungibleTokenInfo{"verify-transfer-token-ownership", "", "", "", "TNFT-PUBLIC-FALSE-02", "nft-mostafa", "nft-grp-addr-2", "token metadata", "", "", "", true, false, true, true, false, "nft-jeansoon", "0", "nft-carlo", "", "APPROVE_TRANFER_TOKEN_OWNERSHIP", "", "", []string{""}}, "[case-2.0] VERIFY nonfungible token transfer ownership [TNFT-PUBLIC-FALSE-02] - Happy path", nil},
```

Comment:
Only for approve_create_token and approve_transfer_ownership we have to wait.

Reply:
Will test on this and revert.

Reply:
Please see below latest update.